src/Timeline.js

```diff
@@ -196,10 +222,35 @@ export default class Timeline extends Axis {
       @chainable
   */
   render(callback) {
+    const {height, y} = this._position;
+    this._d3Scale = scaleTime().domain(this._domain.map(date)).range([0, this._width]);
```

Comment:
1. don't save this to `this`, as it's only used in these few lines
2. this scale/measuring logic should only by run if buttonBehavior is "auto" and `this._ticks` is defined
3. the domain of the scales needs to be `this._ticks`

Reply:
1: ✅ 
2: ✅ 
3: Working in it